src/main/java/edu/nps/moves/dis7/Beam.java

```diff
@@ -0,0 +1,339 @@
+package edu.nps.moves.dis7;
+
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.Serializable;
+import java.util.Objects;
+
+
+/**
+ * This field shall specify information about a particular active beam. Section 7.6.2.
+ *
+ * Copyright (c) 2008-2016, MOVES Institute, Naval Postgraduate School. All rights reserved. This work is licensed under the BSD open source
+ * license, available at https://www.movesinstitute.org/licenses/bsd.html
+ * @author DMcG
+ */
+public class Beam extends Object implements Serializable
```

Comment:
I think we should rename the class to `ElectronicEmissionBeamData` so that it matches the class name for DIS 6.

https://github.com/open-dis/open-dis-java/blob/master/src/main/java/edu/nps/moves/dis/ElectronicEmissionBeamData.java

Reply:
Done